snapshot-labs/sx-monorepo
# fix(ui): fix missing choice in approval voting votes results list

### Summary

While merging the approval voting support PR (https://github.com/snapshot-labs/sx-monorepo/pull/178), a detail was missed: the votes list (both in modal and tab) are only showing the first choice, whereas a user can vote for multiple choices.

This PR fix this issue, and will show all the different choices chosen by the user

### How to test

1. Go to an approval voting (e.g. http://localhost:8080/#/s:pistachiodao.eth/proposal/0x08c3bd2960700525770a1d634f8599ba967e55fcc05b6c1649d984d88253769d/votes)
2. It should list all the choices chosen by the voter
3. Open the votes modal 
4. It should also list all the choices chosen by the user

All choices now have a tooltip, in case the original text is truncated
